packages/office-ui-fabric-react/src/components/FloatingPicker/PeoplePicker/examples/FloatingPeoplePicker.Basic.Example.tsx

```diff
@@ -9,129 +8,100 @@ import {
 } from 'office-ui-fabric-react/lib/FloatingPicker';
 import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
 import { people } from '@uifabric/example-data';
+import { useConst } from '@uifabric/react-hooks';
+
+const suggestionsStore = new SuggestionsStore<IPersonaProps>();
```

Comment:
This needs to be in a useConst inside the component, so you're not sharing state between instances.
`const suggestionsStore = useConst(() => new SuggestionsStore<IPersonaProps>());`